horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/data/Access.java

```diff
@@ -35,4 +35,7 @@ public static Access fromString(String str) {
          return Access.valueOf(str);
       }
    }
+   public static Access fromInt(int a) {
+      return VALUES[a];
+   }
```

Comment:
In the API, I think we want to expose this as a String, and not an Integer. 
For example, looking at the output from: https://horreum.hyperfoil.io/openapi/#operation/DatasetService_listByTest

```
"owner": "performance-team",
"access": "PUBLIC",
"view": "string",
```
makes more sense for a user than;

```
"owner": "performance-team",
"access": 0,
"view": "string",
```